torch/csrc/jit/python/init.cpp

```diff
@@ -694,6 +694,29 @@ void initJITBindings(PyObject* module) {
             std::tie(data, size) = self.getRecord(key);
             return py::bytes(reinterpret_cast<const char*>(data.get()), size);
           })
+      .def(
+          "get_storage_from_record",
+          [](PyTorchStreamReader& self,
+             const std::string& key,
+             size_t numel,
+             py::object dd) {
```

Comment:
nit - why numel and dd? I see `py::object` usually called obj or buffer in other places.